opentelemetry-sdk/src/opentelemetry/sdk/_metrics/view.py

```diff
@@ -0,0 +1,154 @@
+                version the instrument meter must have to match the view.
+
+            meter_schema URL : This is an instrument matching attribute: the
+                schema URL the instrument meter must have to match the view.
+
+            name: This is a metric stream customizing attribute: the name of
+                the metric stream. If `None`, the name of the instrument will
+                be used.
+
+            description: This is a metric stream customizing attribute: the
+                description of the metric stream. If `None`, the description of
+                the instrument will be used.
+
+            attribute_keys: This is a metric stream customizing attribute: this
+                is a set of attribute keys. If not `None` then only the
+                measurement attributes that are in `attribute_keys` will be
+                used to identify the metric stream.
+
+            aggregation: This is a metric stream customizing attribute: every
+                metric stream has an aggregation instance, this is the class of
+                aggregation this instance will be. If `None` the default
+                aggregation class of the instrument will be used.
+        """
+
+        if (
+            instrument_type
+            is instrument_name
+            is meter_name
+            is meter_version
+            is meter_schema_url
+            is None
+        ):
+            raise Exception(
+                "Some instrument selection "
+                "criteria must be provided for View {name}"
+            )
+
+        self._name = name
+        self._instrument_type = instrument_type
+        self._instrument_name = instrument_name
+        self._meter_name = meter_name
+        self._meter_version = meter_version
+        self._meter_schema_url = meter_schema_url
+
+        self._description = description
+        self._attribute_keys = attribute_keys
+        self._aggregation = aggregation
+
+        if self._name is not None:
+            self._previously_matched_instrumentation_infos = set()
+
+    # pylint: disable=too-many-return-statements
+    # pylint: disable=too-many-branches
+    def _match(self, instrument: Instrument) -> bool:
+
+        if self._instrument_type is not None:
+            if not isinstance(instrument, self._instrument_type):
+                return False
+
+        if self._instrument_name is not None:
+            if not fnmatch(instrument.name, self._instrument_name):
+                return False
+
+        if self._meter_name is not None:
+            if not fnmatch(
+                instrument.instrumentation_info.name, self._meter_name
+            ):
+                return False
+
+        if self._meter_version is not None:
+            if not fnmatch(
+                instrument.instrumentation_info.version, self._meter_version
+            ):
+                return False
+
+        if self._meter_schema_url is not None:
+            if not fnmatch(
```

Comment:
Why are we using `fnmatch` for schema_url?

Reply:
Fixed